release/config/crd/bases/anywhere.eks.amazonaws.com_bundles.yaml

```diff
@@ -172,6 +201,9 @@ spec:
                               type: string
                           type: object
                         kubeProxy:
+                          description: |-
+                            Image represents a container image asset along with metadata such as OS,
```

Comment:
Looks like generic descriptions got added for various fields like Image,Manifest. Was this intended?

Reply:
Yes, my problem is that they are very generic and doesn't describe the field effectively and can be confusing too for end users. I guess controller-gen started picking up comment on the type definition if no comment exists on the field. I think we should either add relevant comment/descriptions for fields or get rid of these by using a different controller-gen version or some configuration for now.